ts/sagas/identification.ts

```diff
@@ -86,7 +87,10 @@ function* waitIdentificationResult(): Generator<
 
     case getType(identificationSuccess): {
       // if the identification has been successfully, check if the current session is still valid
-      yield* put(checkCurrentSession.request());
+      const isFastLoginEnabled = yield* select(isFastLoginEnabledSelector);
+      if (!isFastLoginEnabled) {
+        yield* put(checkCurrentSession.request());
+      }
       return IdentificationResult.success;
```

Comment:
This is necessary, because with fast login feature the check of the current session is done in other places. Leaving this check here too could lead to a token refresh loop we want to avoid.